http/cookie.ts

```diff
@@ -1,11 +1,103 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-import { ServerRequest } from "./server.ts";
+import { ServerRequest, Response } from "./server.ts";
+import { assert } from "../testing/asserts.ts";
+import { pad } from "../strings/pad.ts";
 
 export interface Cookie {
   [key: string]: string;
 }
 
-/* Parse the cookie of the Server Request */
+export interface CookieValue {
+  name: string;
+  value: string;
+}
+
+export interface CookieOptions {
```

Comment:
I think this should be called "Cookie" 

https://golang.org/pkg/net/http/#Cookie

Reply:
Refactored